sdk/core/core-lro/package.json

```diff
@@ -108,7 +109,15 @@
     "vitest": "^2.0.5"
   },
   "//metadata": {
-    "migrationDate": "2023-03-08T18:36:03.000Z"
+    "migrationDate": "2023-03-08T18:36:03.000Z",
+    "sampleConfiguration": {
+      "skipFolder": true,
```

Comment:
nit: it would be really cool if we can get this samples running - how difficult would that be?

Reply:
Ah that was by mistake, it works. Addressed in https://github.com/Azure/azure-sdk-for-js/pull/32469/commits/9babca3eabce977777fdbb2aba1959735d48fcd2